cmdstanpy/utils.py

```diff
@@ -84,38 +84,43 @@ def get_latest_cmdstan(cmdstan_dir: str) -> Optional[str]:
     """
     Given a valid directory path, find all installed CmdStan versions
     and return highest (i.e., latest) version number.
-    Assumes directory populated via `install_cmdstan`.
+
+    Assumes directory consists of CmdStan releases, created by
+    function `install_cmdstan`, and therefore dirnames have format
+    "cmdstan-<maj>.<min>.<patch>" or "cmdstan-<maj>.<min>.<patch>-rc<num>",
+    which is CmdStan release practice as of v 2.24.
```

Comment:
for the record, conda builds do not do this. They end up installed in a directory which has the top level name of 'cmdstan'. We can change this going forward, but I didn't realize we relied on the directory names so existing installations will not reflect this. I don't think this is a big deal because I'm pretty sure get_latest_cmdstan is never called when `$CMDSTAN` is set

Reply:
correct.

Reply:
great, just wanted to check